doc/release/release_dev.rst

```diff
@@ -47,6 +47,8 @@ API Changes
   and ``skimage.transform.rescale`` has been set to ``reflect``.
 - Default value of ``anti_aliasing`` parameter in ``skimage.transform.resize``
   and ``skimage.transform.rescale`` has been set to ``True``.
+- Removed the ``skimage.test`` function. This functionality can be acheived
+  by calling ``pytest`` directly.
```

Comment:
I understand the sentiment, but I think we don't want to do this. The reason this exists is that we used to use nosetests and were bitten when it reached end-of-life. `skimage.test` allows us to keep the same testing syntax even if we decide to switch frameworks later on.

Reply:
also, "achieved". =P

I'm neither +1 or -1 on this btw. Running individual tests is more natural if we normalise with pytest. Just wanted to flag it since others (@soupault) have in the past preferred strongly hiding pytest (using skimage.testing.raises instead of pytest.raises, for example).

Reply:
@jni, I understand that the idea was to have the testing localized, but after having going through a few of the other build procedures, it seems that `skimage.test` was never used in the first in place.

When I tried `skimage.test` myself a few months back, I thought I wasn't using the function correctly, but a recent PR showed me that it wasn't just me.

I'm glad to go back to the old functionality, but I feel like the ship has sailed on that, even before my PR. I didn't have to change any of the other tests when I removed this function.